src/main/java/org/terasology/minion/move/SetupContinuousTravelNode.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2018 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.minion.move;
+
+import org.terasology.logic.behavior.BehaviorAction;
+import org.terasology.logic.behavior.core.Actor;
+import org.terasology.logic.behavior.core.BaseAction;
+import org.terasology.logic.behavior.core.BehaviorState;
+import org.terasology.navgraph.WalkableBlock;
+import org.terasology.pathfinding.model.Path;
+
+/**
+ * Essential setup for a character to travel along a determined path. This is exported from {@link MoveAlongPathNode#construct(Actor)}, which allows for characters to
```

Comment:
Just a comment: I try to stick to max. 120 chars per line, if possible, so I don't have to scroll horizontally. GitHub shows 130 chars, eclipse uses 120 by default afaik. Especially for block comments it should be easy to do, but it's a matter of personal preference, I guess.

Reply:
I transferred these nodes to Terasology/Behaviors#22, and I wrapped the comments to less than 120 characters. I've added a line guide to IDEA so all lines will be less than 120 characters in the future 👍